app/views/products/_form.html.erb

```diff
@@ -1,25 +1,41 @@
-<% if @product.errors.any? %>
-  <p>Please fix the following...</p>
-  <ul>
-    <% @product.errors.each do |error, msg|%>
-      <li><%= error %>: <%= msg %></li>
-    <% end %>
-  </ul>
-<% end %>
-<%= form_with model: @product do |f| %>
-  <%= f.label :name %>
-  <%= f.text_field :name %>
-  <%= f.label :category %>
-  <%= f.select :category, [['Shoes'], ['Clothing'], ['Household'], ['Clothing']]%>
-  <%= f.label :stock %>
-  <%= f.text_field :stock %>
-  <%= f.label :price %>
-  <%= f.text_field :price %>
-  <%= f.label :retired %>
-  <%= f.select :retired, [['Active'], ['Retired']]%>
-  <%= f.label :description %>
-  <%= f.text_area :description %>
-  <%= f.label :photo_url %>
-  <%= f.text_field :photo_url %>
-  <%= f.submit submit_button_label%>
-<% end %>
+<section class="container">
+  <% if @product.errors.any? %>
+    <p>Please fix the following...</p>
+    <ul>
+      <% @product.errors.each do |error, msg|%>
+        <li><%= error %>: <%= msg %></li>
+      <% end %>
+    </ul>
+  <% end %>
+  <%= form_with model: @product do |f| %>
+    <div class="form-group">
+      <%= f.label :name %>
+      <%= f.text_field :name, class: "form-control" %>
+    </div>
+    <div class="form-group">
+      <%= f.label :categories %>
+      <%= f.collection_select :category_ids, Category.order(:label), :id, :label, {}, {multiple: true}%>
+    </div>
+    <div class="form-group">
+      <%= f.label :stock %>
+      <%= f.text_field :stock, class:"form-control"%>
+    </div>
+    <div class="form-group">
+      <%= f.label :price %>
+      <%= f.text_field :price, class:"form-control"%>
+    </div>
+    <div class="form-group">
+      <%= f.label :retired %>
+      <%= f.select :status, [['Retired'], ['Active']], {},{class: "form-control"} %>
```

Comment:
if a user adds a new product, we can assume the product will be Active so I think Active should be first before Retired